src/ДЗ. TypeScript. Примітивні літеральні типи та типізація обʼєктів.ts

```diff
@@ -1,188 +1,168 @@
-<<<<<<< HEAD
 class School {
   // implement 'add area', 'remove area', 'add lecturer', and 'remove lecturer' methods
 
-  _areas = [];
-  _lecturers = []; // Name, surname, position, company, experience, courses, contacts
+  _areas: string[] = [];
+  _lecturers: (string | number)[] = []; // Name, surname, position, company, experience, courses, contacts
 
-  get areas() {
-    this._areas;
+  get areas(): typeof this._areas {
+    return this._areas;
   }
 
-  get lecturers() {
-    this._lecturers;
+  get lecturers(): typeof this._lecturers {
+    return this._lecturers;
   }
-}
 
-class Area {
-  // implement getters for fields and 'add/remove level' methods
```

Comment:
Can't find getters for all fields. Please, double check it.

Reply:
fixed